portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,88 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Shreya Barua Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
+    <ul id = "navbar">
+    <li id = "heading"><a href="#contact">Contact</a></li>
+    <li id = "heading"><a href="#cooking">Cooking</a></li>
+    <li id = "heading"><a href="#projects">Projects</a></li>
+    <li id = "heading"><a href="#default">About Me</a></li>
```

Comment:
Please use `class` instead of `id` for these. `id` should be used elements that are unique on a page.

Though also, if you are leaving the CSS for follow-up, you can leave the new classes and ids out of this PR.

Reply:
I am making a separate PR for the Nav Bar which this is a part of but I will definitely make those changes on that PR.

Reply:
Sorry for some confusion I think. I meant that you could remove the `id = "header"` attribute from this PR since it's not getting used anyway. It would be good for the code to be in a good state whenever it's in master, so please either remove the `id` or change to `class`.